package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "@types/validator",
+  "version": "0.0.1",
```

Comment:
if we add one, it should be like with `@types`, where the major and minor matches the source library and patch is for the typings version